index.html

```diff
@@ -148,6 +148,43 @@ <h1>Multi-Post Stories</h1>
  
             </article>
         </section>
+        <section class="contact-form">
+            <h3>CONTACT US</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p> 
+            <form class="form" action="https://formspree.io/f/xoqrbwdq" method="POST">
+                <div>
+                    <label for="text">
+                        <input type="text" id="text" placeholder="Full Name" required minlength="6" maxlength="30">
+                    </label>
+                </div>
+                <div>
+                    <label for="email">
+                        <input type="email" id="email" placeholder="yannickmugisha23@gmail.com" required>
+                    </label>
+                    
+                </div>
+                <div>
+                    <label for="textarea">
+                        <textarea name="textarea" cols="50" rows="8" placeholder="Enter text here" required minlength="15" maxlength="500"></textarea>
+                    </label>
+                    
+                </div>
+                <div class="down-form">
+                    <button>See project</button>

```

Comment:
[OPTIONAL] I think it would be better to stick to the design and use "Send" as submit button text instead of "See project"
<img width="214" alt="Screenshot 2023-01-18 at 9 33 21 PM" src="https://user-images.githubusercontent.com/25887253/213265556-d8e22b1a-52b8-442c-b2f3-554209a734f1.png">